doc/sphinx/administration_portal/brand/clients/retail.rst

```diff
@@ -0,0 +1,136 @@
+.. _retail_clients:
+
+******
+Retail
+******
+
+Retail clients are even a more lightweight client type than *Residential clients*.
+
+They just provide a SIP trunking service that include these features:
+
+- Configure one or more retail accounts (SIP devices).
+- Setup one or more DDIs.
+- **Place external calls** showing one of those DDIs.
+- **Receive external calls** to their DDIs.
+- Record calls.
+
+.. warning:: No users, no extensions, no internal calls, no huntgroups, no IVRs, no voicemail, no call forwardings...
+             just **incoming and outgoing external calls**.
+
+.. error:: Retail clients and their accounts **MUST use Brand's SIP domain in their SIP messages**.
+
+Differences between retail and residential clients
+--------------------------------------------------
+
+There is an important key difference between these two clients: **retail client's call do not traverse
```

Comment:
_retail client's call do**es** not traverse_

 or

 _retail client call**s** do not traverse_